### PR TITLE
Add rat cams (add new coax + control unit types)

### PR DESCRIPTION
**Pull request by Echoskope**
Added the new rat cam camera controller and sensors. Created a new module, "Camera Control Unit", and line type, "Coax".

### CODE REVIEW

File: apps/website/src/data/tech/network.ts [hunk 1]
```diff
@@ -372,11 +372,50 @@ const data: RootNetworkItem[] = [
...
-                type: "camera",
-                name: "Chinchilla",
-                model: "AXIS M1065-L (Fixed)",
-                url: "https://www.axis.com/products/axis-m1065-l/",
-                connection: { type: "ethernet", location: "wall" },
+                type: "switch",
+                name: "Rat/Chin Switch",
+                model: "Ubiquiti USW-Lite-8-PoE",
+                url: "https://store.ui.com/us/en/pro/category/switching-utility/products/usw-lite-8-poe",
+                connection: { type: "ethernet", location: "buried" },
```
Comment (MattIPv4) [inline apps/website/src/data/tech/network.ts:379]: 🤔 The ethernet run to this switch is buried?

Comment (Echoskope) [reply apps/website/src/data/tech/network.ts:379]: Good catch, updated to wall.

File: apps/website/src/components/tech/Network.tsx [hunk 2]
```diff
@@ -79,6 +79,10 @@ const nodeTypes: {
     container: "border-yellow-700",
     eyebrow: { name: "Server", color: "text-yellow-700" },
   },
+  controlunit: {
+    container: "border-red-700",
+    eyebrow: { name: "Camera Control Unit", color: "text-yellow-700" },
```
Comment (MattIPv4) [inline apps/website/src/components/tech/Network.tsx:84]:
```suggestion
    eyebrow: { name: "Camera Control Unit", color: "text-red-700" },
```
I would think we'd want the text to match the border, to be consistent with everything else?

File: apps/website/src/data/tech/network.ts [hunk 3]
```diff
@@ -372,11 +372,50 @@ const data: RootNetworkItem[] = [
...
+                  {
+                    type: "controlunit",
+                    name: "Rat Control Module",
+                    model: "AXIS F9114",
+                    url: "https://www.axis.com/products/axis-f9114-main-unit/",
+                    connection: { type: "ethernet", location: "wall" },
+                    links: [
+                      {
+                        type: "camera",
+                        name: "Rat (Top Level)",
```
Comment (MattIPv4) [inline apps/website/src/data/tech/network.ts:397]:
```suggestion
                        name: "Rat Upper",
```

File: apps/website/src/data/tech/network.ts [hunk 4]
```diff
@@ -372,11 +372,50 @@ const data: RootNetworkItem[] = [
...
+                        model: "AXIS F4105-LRE",
+                        url: "https://www.axis.com/products/axis-f4105-lre-dome-sensor/",
+                        connection: { type: "coax", location: "wall" },
+                      },
+                      {
+                        type: "camera",
+                        name: "Rat (Middle Level)",
```
Comment (MattIPv4) [inline apps/website/src/data/tech/network.ts:404]:
```suggestion
                        name: "Rat Middle",
```

File: apps/website/src/data/tech/network.ts [hunk 5]
```diff
@@ -372,11 +372,50 @@ const data: RootNetworkItem[] = [
...
+                        model: "AXIS F4105-LRE",
+                        url: "https://www.axis.com/products/axis-f4105-lre-dome-sensor/",
+                        connection: { type: "coax", location: "wall" },
+                      },
+                      {
+                        type: "camera",
+                        name: "Rat (Bottom Level)",
```
Comment (MattIPv4) [inline apps/website/src/data/tech/network.ts:411]:
```suggestion
                        name: "Rat Lower",
```

**Review by MattIPv4 — COMMENTED** [review]
Thoughts on this for the cam naming, to be consistent with how we named the upper/lower switches?

**Review by MattIPv4 — APPROVED** [review]

**Pull request merged**